test/llvm/abi/export-vs-c.compopts

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+FNS=`cat export-vs-c.chpl | grep --only-matching -e '[a-zA-Z0-9_]*_\(c\|chapel\)(' | sed 's/(//g' | sort -u`
+FNS=`echo $FNS | sed 's/ /,/g'`
+
+echo --ccflags -fdiscard-value-names --llvm --llvm-print-ir-stage basic --llvm-print-ir $FNS
```

Comment:
I think this should have a comment saying that the base version of this file is not expected to be useful due to the .notest but the symlinked versions are, or something along those lines

Reply:
Though really if we don't have other subdirectories for this test, I'm not sure I entirely see the point, since it won't end up being run in other places

Reply:
Oh, it probably wasn't obvious, but I intend to add an arm subdirectory next to the x86-64 one. However I want to do that in a future PR.

Reply:
That seems reasonable.  I think a comment on the .compopts would still be warranted, to avoid someone in the future looking at the directory and removing either the .compopts or the .notest out of confusion.